File: KCD/HMMasterMissionCommand.m
//
//  HMMasterMissionCommand.m
//  KCD
//
//  Created by Hori,Masaki on 2014/02/17.
//  Copyright (c) 2014年 Hori,Masaki. All rights reserved.
//

#import "HMMasterMissionCommand.h"

@implementation HMMasterMissionCommand
- (NSString *)dataKey
{
	return @"api_data.api_mst_mission";
}
- (void)execute
{
	[self commitJSONToEntityNamed:@"MasterMission"];
}
@end

File: KCD/HMMasterMissionCommand.swift
//
//  HMMasterMissionCommand.swift
//  KCD
//
//  Created by Hori,Masaki on 2015/01/12.
//  Copyright (c) 2015年 Hori,Masaki. All rights reserved.
//

import Cocoa

class HMMasterMissionCommand: HMJSONCommand {
	override var dataKey: String {
		return "api_data.api_mst_mission"
	}
	override func execute() {
		super.commitJSONToEntityNamed("MasterMission")
	}
}
